src/Symfony/Component/Routing/Annotation/Route.php

```diff
@@ -62,6 +72,16 @@ public function getPath()
         return $this->path;
     }
 
+    public function setLocales(array $locales)
+    {
+        $this->locales = $locales;
+    }
+
+    public function getLocales(): array
```

Comment:
missing phpdoc. it's not clear what this method returns at all. a better name would also be `getLocalizedPaths` because that is what it seems to return

Reply:
esp. if we want to add to possiblity to add localized hosts as well

Reply:
updated in 7101893